### PR TITLE
Camera Obscura pictures now steal souls

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
ghost-hunters-tech-guide-ghost-camera

But only for the `MOB_SPIRIT` biotype.  This will also stun and reveal revenants briefly even if they are hidden.

## Why It's Good For The Game
It's funny and a slightly silly way to combat revenants by using paparazzi.   

## Changelog
:cl:
balance: "Camera Obscura" (crafted with holy water + camera) will now steal the soul of ghostly apparitions when a picture is taken causing damage.  This will also stun and reveal revenants briefly. 
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
[latest emergent revenant gameplay](https://www.youtube.com/watch?v=UaSOAmRkY5w)

File: code/modules/photography/camera/camera.dm [hunk 1]
```diff
@@ -210,6 +210,15 @@
 			dead_spotted += mob
 		desc += mob.get_photo_description(src)
 
+		if(mob.mob_biotypes & MOB_SPIRIT) // time to steal your soul
```
Comment (LemonInTheDark) [inline code/modules/photography/camera/camera.dm:213]: test your code and I'll reopen this

**Comment by LemonInTheDark** [comment]
ah you know what no that's dumb

**Comment by Likteer** [comment]
@timothymtorres Hi, there is an item called "camera obscura" that can take pictures of ghosts, which are used for making hauntium. Maybe you'd like to use that? The recipe is camera+holy water, if I recall.

**Comment by RaveRadbury** [comment]
New Fatal Frame looking legit

**Comment by Fikou** [comment]
> dreamseeker_H8ptYCD9aT [ dreamseeker_H8ptYCD9aT ](https://user-images.githubusercontent.com/5195984/230412925-2dab59b8-885e-409f-8732-ee4bfd486735.gif) [ ](https://user-images.githubusercontent.com/5195984/230412925-2dab59b8-885e-409f-8732-ee4bfd486735.gif)
> 
> I think the DNM is overkill considering that I had this PR in a draft state. This allowed me to type up the PR information and do some simple CI testing before I got home saving development time.

can you fix cameras being bugged

**Comment by ATH1909** [comment]
can you stunlock a revenant to death with this

**Comment by Vect0r2** [comment]
> can you stunlock a revenant to death with this by cycling between 4 camera obscuras?
> 
> if so, an AoE hitscan touch of death combo sounds really cancerous to deal with as a revenant.

Know what else is canerous? Revenat.

**Comment by YakumoChen** [comment]
Creditable ranged stuns are gay
It should reveal revanent but not stun it

**Comment by Redrover1760** [comment]
> 

I mean, the coordination it would take for four people with cameras to perfectly time the stuns and pictures to actually permastun a revenant would be pretty hard, but some kind of immunity to being stunned after being stunned from a camera obscura would probably be a good failsafe regardless. Does it reveal a revenant if its already revealed?

**Comment by MercuryArrow** [comment]
Honestly quite cool - I've joked a few times about having the camera obscura be made into a functional anti-spirit-mob weapon a few times to properly befit its inspiration so it's nice seeing someone else had the same idea.  Now all it needs is a better sprite.

**Comment by YakumoChen** [comment]
> > 
> 
> I mean, the coordination it would take for four people with cameras to perfectly time the stuns and pictures to actually permastun a revenant would be pretty hard, but some kind of immunity to being stunned after being stunned from a camera obscura would probably be a good failsafe regardless. Does it reveal a revenant if its already revealed?

The best case scenario would be to snap the picture while it's already revealed and persued, stunning it and dealing more damage while people are already trying to wail it. I can imagine that'll be more than enough to finish it off more reliably.

**Comment by optimumtact** [comment]
Add an exponential cool down on the stun so you can't permalock reveanents with this and then it's mergeable.

**Comment by ZephyrTFA** [comment]
The stun period is now 2 seconds, which means you would need to coordinate a group of eight people, all with camera obscuras. The amount of cohesion and coordination this would take would never happen; and if it does happen, the station deserves the W

**Comment by ZephyrTFA** [comment]
Aww, read that wrong. You removed chain stunning entirely. While more balanced still makes me sad.

**Pull request merged**